src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+
+  if (days >= 7) {
+    return days * dayPrice - 50;
+  }
+
+  if (days >= 3) {
+    return days * dayPrice - 20;
```

Comment:
Replace the '3' and '20' with constants that describe these values. For example, 'SHORT_TERM_RENTAL' and 'SHORT_TERM_DISCOUNT'.